python/MooseDocs/extensions/MooseBibtex.py

```diff
@@ -26,6 +28,26 @@ class MooseBibtex(MooseCommonExtension, Preprocessor):
   def __init__(self, markdown_instance=None, **kwargs):
     MooseCommonExtension.__init__(self, **kwargs),
     Preprocessor.__init__(self, markdown_instance)
+    self.macro_files = kwargs['macro_files']
```

Comment:
We generally don't use "public" attributes for MOOSE code, also you should also provide a default in-case the file is not provided.
```
self._macro_files = kwargs.pop('macro_files', None)
```